lxc/action.go

```diff
@@ -239,6 +239,10 @@ func (c *cmdAction) doAction(action string, conf *config.Config, nameArg string)
 			return err
 		}
 
+		if shared.IsFalse(current.ExpandedConfig["security.protection.start"]) {
```

Comment:
The validation should not happen on the client side. Instead the server has to check if the instance (VM or container) has `security.protection.start=false` set.